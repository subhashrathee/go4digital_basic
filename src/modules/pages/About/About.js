import React from 'react'
import Body from '../../common/Body/Body'
import Footer from '../../common/Footer/Footer'
import JSONContent from '../../config/content/about_us.js' 

//Content passed into body component
const content = JSONContent.about_us.join('');

/**
* About the class page component used to render the page.
*
* Props:      None
* 
* State vars: None
*/
class About extends React.Component {
  render () {
    return (
      <div>
        <div  className="hero-image-about"></div>
        <Body heading="About the class" content={content} />
        <Footer />
      </div>
    )
  }
}

export default About;
