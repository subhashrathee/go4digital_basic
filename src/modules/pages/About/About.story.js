import React from 'react';
import { storiesOf } from '@storybook/react';
import { action } from '@storybook/addon-actions';
import { BrowserRouter } from 'react-router-dom'
import About from './About'

storiesOf('Pages', module)
  .add('About', () => (
      <About />
  ));
