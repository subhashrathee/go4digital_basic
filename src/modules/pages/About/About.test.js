import React from 'react';
import About from './About';
import renderer from 'react-test-renderer';
import { BrowserRouter, Link } from 'react-router-dom'

test('About component', () => {
  const component = renderer.create(
    <About />
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
