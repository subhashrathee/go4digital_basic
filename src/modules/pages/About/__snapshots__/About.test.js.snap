// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About component 1`] = `
<div>
  <div
    className="hero-image-about"
  />
  <div
    className="main"
  >
    <h1>
      About go4digital
    </h1>
    <p>
      go4digital is passionate about offering high quality digital services to small and medium businesses at an affordable cost. We believe in providing level playing field to these businesses who can only afford workable solutions in the current IT landscape 
    </p>
    <p>
      Rest assured, we would like to grow with you and put all efforts in helping your business to grow in digital world. Though we have lot of expertise within the team, we are starting our digital journey now and would like to travel this journey with you and share our successes and failures with you. 
    </p>
    <p>
      From a small website development company to an enterprise grade software solutions, from simple bulk emailing to sophisticated digital campaign, we are up for challenge.
    </p>
    <p>
      We believe in iterative enhancements. If you have passion for your business to succeed, we would like to offer partnership at every stage of your business journey.
    </p>
    <p>
      Right from building blazing fast responsive websites, converting an existing website into today's age website, we offer digital marketing services to help build brand reputation of your business in digital world.
    </p>
  </div>
  <footer>
    <p>
      Â© 
      2018
       go4digital. All rights reserved.
    </p>
  </footer>
</div>
`;
